community-meetings/2021-09-14.md

```diff
@@ -0,0 +1,22 @@
+# Flatcar community call Tuesday, 14th of September, 5:30 pm CEST
```

Comment:
Is the time zone CEST or CEDT? It would be good to also list the local time in GMT, EDT, PDT for a global audience.

Reply:
That's a good point, will add.